.github/workflows/ci.yml

```diff
@@ -78,9 +75,6 @@ jobs:
         run: Scripts/github/prepare-simulators.sh ${{ matrix.platforms }}
       - name: Build and Test Framework
         run: Scripts/build.swift ${{ matrix.platforms }}
-      - name: Upload Coverage Reports
-        if: success()
-        run: Scripts/upload-coverage-reports.sh ${{ matrix.platforms }}
```

Comment:
Similarly, I was having trouble getting Xcode 12 exporting `lcov` file reports, so I deleted the uploads rather than investing more time here. We don't need coverage on all Xcode runs.

Reply:
Asking to learn: if we upload multiple coverage reports for the same code like we were doing before, what does that mean in codecov? Does codecov take the union?

Reply:
Yeah codecov unions (they call it "merging") them: https://docs.codecov.com/docs/merging-reports